common/src/androidTest/java/com/microsoft/identity/common/adal/internal/util/HashMapExtensionTests.java

```diff
@@ -61,7 +61,7 @@ public void testURLFormDecodeNegative() throws IllegalArgumentException,
         final String methodName = "urlFormDecode";
         Object object = ReflectionUtils.getNonPublicInstance("com.microsoft.identity.common.adal.internal.util.HashMapExtensions");
         Method m = ReflectionUtils.getTestMethod(object, methodName, String.class);
-        HashMap<String, String> result = (HashMap<String, String>) m.invoke(object, "nokeyvalue");
+        HashMap<String, String> result = (HashMap<String, String>) m.invoke(object, "");
```

Comment:
Is this worth commenting that this checks for 'CertAuthorities='?

Reply:
Resolved -- see below